pom.xml

```diff
@@ -348,10 +347,16 @@ under the License.
 		-->
 		<dependencies>
 
+			<dependency>
+				<groupId>org.apache.flink</groupId>
+				<artifactId>flink-shaded-force-shading</artifactId>
+				<version>${flink.shaded.version}</version>
```

Comment:
added management entry because we're running into convergence issues with teh ES connector pulling in 15.0.
We'll need to figure this out for force-shading eventually.